dev-docs/modules/contxtfulRtdProvider.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: page_v2
+title: Contxtful RTD Provider  
+display_name: Contxtful RTD Provider  
+description: Activates Receptivity to evaluate ad success in real-time by focusing on attention and context.
+page_type: module
+module_type: rtd
+module_code : contxtfulRtdProvider
+enable_download : true
+vendor_specific: true
+sidebarType : 1
+---
+
+# Contxtful Real Time Data Module
+
+## Overview
+
+The Contxtful RTD module offers a unique feature—Receptivity. Receptivity is an efficiency metric, enabling the qualification of any instant in a session in real time based on attention. The core idea is straightforward: the likelihood of an ad’s success increases when it grabs attention and is presented in the right context at the right time.
+
+## Integration
+
+To incorporate this module into your `prebid.js`, compile the module using the following command:
+
+```sh
+gulp build --modules=contxtfulRtdProvider,<other modules...>
+```
+
+### Configuration
+
+Configure the `contxtfulRtdProvider` by passing the required settings through the `setConfig` function in `prebid.js`.
+
+```js
+import pbjs from 'prebid.js';
+
+pbjs.setConfig({
+  "realTimeData": {
+    "auctionDelay": 1000,
+    "dataProviders": [
+      {
+        "name": "contxtful",
+        "waitForIt": true,
+        "params": {
+          "version": "<API Version>",
+          "customer": "<Contxtful Customer ID>"
+        }
+      }
+    ]
+  }
+});
+```
+
+#### Parameters
+
+| Name       | Type     | Scope    | Description                               |
+|------------|----------|----------|-------------------------------------------|
+| `version`  | `string` | Required | Specifies the API version of Contxtful.   |
+| `customer` | `string` | Required | Your unique customer identifier.          |
```

Comment:
```suggestion

{: .table .table-bordered .table-striped }
| Name       | Type     | Scope    | Description                               |
|------------|----------|----------|-------------------------------------------|
| `version`  | `string` | Required | Specifies the API version of Contxtful.   |
| `customer` | `string` | Required | Your unique customer identifier.          |
```

This makes the table look like a table

Reply:
Thank you for the great feedback! I applied your suggestion.

Reply:
Good day @muuki88 !
Is this change adequate for you? I let you resolve the thread if so; otherwise let me know I'm available to make fixes.
Thanks!